### PR TITLE
Adding dependencies in the requirements.txt for hanhu data

### PR DESCRIPTION
**Pull request by imamnurby**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -28,3 +28,31 @@ wordcloud
...
+librosa==0.10.1
+mechanize==0.4.9
+prettytable==3.10.0
+pycryptodome==3.14.1
+python_http_client==3.3.7
+Requests==2.31.0
+requests_mock==1.11.0
+rsa==4.9
+scikit_learn
+scipy
```
Comment (terryyz) [inline requirements.txt:50]: duplicated

File: requirements.txt [hunk 2]
```diff
@@ -28,3 +28,31 @@ wordcloud
...
+keras
+librosa==0.10.1
+mechanize==0.4.9
+prettytable==3.10.0
+pycryptodome==3.14.1
+python_http_client==3.3.7
+Requests==2.31.0
+requests_mock==1.11.0
+rsa==4.9
+scikit_learn
```
Comment (terryyz) [inline requirements.txt:49]: invalid + duplicated

**Comment by terryyz** [comment]
Hi @imamnurby, thanks for the contribution. Please note that there are a few duplicates in the `requirements.txt`. Please double-check and try to `pip install -r requirements.txt` in your local environment or enable GitHub Actions in your forked repo.

Cheers

**Comment by imamnurby** [comment]
I remove the scikit_learn and scipy. I checked the version in my environment, and it matches the version of the first occurrence.
Also, I think I include the change in #10, so I will close this one. I apologize.

**Pull request closed without merging**